tests/acceptance/29_simulate_mode/manifest_mode.cf.sub.mustache

```diff
@@ -0,0 +1,66 @@
+body package_module test_module
```

Comment:
This test is what I was thinking of when "as many promise types as possible" was mentioned in the ticket.

But this would be if simulate mode worked with ALL promise types. At this time as I understand it, only files promises are affected by simulate mode other than risky functions.

Reply:
Correct.